woocommerce/woocommerce-gutenberg-products-block
# Add label to Reviews by Product controls count

Part of #588.
Follow-up of #671.
Depends on https://github.com/woocommerce/woocommerce-admin/pull/2569 being merged and a new version of `@wocommerce/components` being released.

#### Accessibility




### Screenshots



### How to test the changes in this Pull Request:

1. In a new or existing post/page add a _Reviews by Product_ block.
2. Verify the product control shows the reviews count next to the product name in the form _X Reviews_ instead of a single number (see screenshots above).
